configtools.spec

```diff
@@ -4,11 +4,16 @@ Release:     1
 Summary:     The configtools module
 
 License:     GPLv3+
-URL:         http://pypi.python.org/pypi/%{name}
-Source0:     http://pypi.python.org/packages/source/c/%{name}/%{name}-%{version}.tar.gz
+URL:         https://pypi.python.org/pypi/%{name}
+Source0:     https://pypi.python.org/packages/source/c/%{name}/%{name}-%{version}.tar.gz
 Buildarch:   noarch
 
+%if 0%{?rhel} == 7
 BuildRequires:	python2-devel
```

Comment:
Why does this work if we appear to require python3 now?

Reply:
This is dealt with in PR#15 I believe. There were commits from that PR in the tree for this PR, which I have tried to separate out, so now the changes in PR#15 relative to master are separate from the changes in this PR relative to master (except for  some changes in .travis.yml: for that, the changes in this PR should win). I hope that makes it clearer.